debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/FieldSelector.java

```diff
@@ -101,14 +108,23 @@ public FieldSelectorBuilder renameFields(String fullyQualifiedFieldReplacements)
          * @return the filter selector that returns the filter to exclude or rename fields in a document
          */
         public FieldSelector build() {
+            errors = new ArrayList<>();
             List<Path> result = new ArrayList<>();
             parse(fullyQualifiedFieldNames, name -> {
-                String[] nameNodes = parseIntoParts(name, name, length -> length < 3, DOT);
+                String[] nameNodes = parseIntoParts(name, name, length -> length < 3, DOT, errors);
+                if (!errors.isEmpty()) {
+                    LOGGER.error(errors.get(0) + " ( expecting " + QUALIFIED_FIELD_EXCLUDE_LIST_PATTERN + ")");
```

Comment:
Instead of just logging an error message, isn't it that we'd want to return it to Kafka Connect's validation routine? And why is it just logging the first error?